### PR TITLE
Fix command limit checking for PluginSDK called commands.

### PR DESCRIPTION
**Pull request by MiranDMC**
Added support of 0 argument for disabling the limit.

### CODE REVIEW

File: cleo_plugins/DebugUtils/DebugUtils.cpp [hunk 1]
```diff
@@ -183,7 +183,7 @@ class DebugUtils
 
     static bool WINAPI OnScriptProcess(CScriptThread* thread)
     {
-        currScript.Reset();
+        currScript.Begin(thread);
```
Comment (x87) [inline cleo_plugins/DebugUtils/DebugUtils.cpp:186]: this line can be removed. if the script is different we call `Begin` anyway (in ProcessCommand)

File: cleo_plugins/DebugUtils/DebugUtils.cpp [hunk 2]
```diff
@@ -183,7 +183,7 @@ class DebugUtils
 
     static bool WINAPI OnScriptProcess(CScriptThread* thread)
     {
-        currScript.Reset();
```
Comment (x87) [inline cleo_plugins/DebugUtils/DebugUtils.cpp:186]: any idea why `Reset` did not work for plg-sdk script called from ASI?

Comment (MiranDMC) [reply cleo_plugins/DebugUtils/DebugUtils.cpp:186]: Plugin SDK just calls ProcessOneCommand.
https://github.com/DK22Pac/plugin-sdk/blob/master/shared/extensions/ScriptCommands.h#L147

Script is not added to any queue and OnScriptProcess is not called either.

File: cleo_plugins/DebugUtils/DebugUtils.cpp (hunk 1 above)
Comment (MiranDMC) [reply cleo_plugins/DebugUtils/DebugUtils.cpp:186]: Yes but it make sense from general design viewpoint. 
Logic inside ProcessCommand is just patching weird behavior.

File: cleo_plugins/DebugUtils/DebugUtils.cpp (hunk 2 above)
Comment (x87) [reply cleo_plugins/DebugUtils/DebugUtils.cpp:186]: But calling reset from ongameprocess hook didn't seem to help either. Weird. Is ongameprocess called each frame?

Comment (MiranDMC) [reply cleo_plugins/DebugUtils/DebugUtils.cpp:186]: All scripts were processed, last script's command count was not cleared and was above the limit due to *.cs4 mode.
Then during same frame ASI plugin started calling it's commands. So every single fake script from SDK was throwing error.

Comment (x87) [reply cleo_plugins/DebugUtils/DebugUtils.cpp:186]: Aaah, so we counted cs4 instructions towards the limit, but displayed the error for a non-cs4 script. Makes sense now

File: cleo_plugins/DebugUtils/SA.DebugUtils.ini [hunk 3]
```diff
@@ -23,6 +23,6 @@ ColorError = "FF30EEFF"
 ColorDebug = "FFEE30FF"
 ColorSystem = "DDDDDDFF"
 
-; limits between 'wait' commands in script after which it is considered hanging
+; limits peer render frame after which script is considered hanging
```
Comment (x87) [inline cleo_plugins/DebugUtils/SA.DebugUtils.ini:26]: ; maximum instructions per script per frame. Set to 0 to disable the limit.

Comment (MiranDMC) [reply cleo_plugins/DebugUtils/SA.DebugUtils.ini:26]: Added

Comment (x87) [reply cleo_plugins/DebugUtils/SA.DebugUtils.ini:26]: does the parser support inline comments?

can you delete the comment above?

Comment (MiranDMC) [reply cleo_plugins/DebugUtils/SA.DebugUtils.ini]: While parsing numbers INI related APIs just stop after first non numeric character.

**Review by x87 — APPROVED** [review]

**Pull request merged**